changelog/add-4569-capture-from-table

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: dev
+
+Add data layer and UI components
```

Comment:
Given this shows in the public changelog, I think we should make it descriptive

Reply:
My intention was to add a proper changelog in the integration PR. This one is only here to make CI happy. But I can change it if it makes things easier.